vaultclient/src/test/java/com/scality/vaultclient/services/GetAccountServiceTest.java

```diff
@@ -0,0 +1,243 @@
+                .accountName(DEFAULT_ACCOUNT_NAME)
+                .build();
+
+        AccountData response = getAccountMockClient.getAccount(getAccountRequestDTO).getAwsResponse();
+
+        assertEquals(DEFAULT_EMAIL_ADDR, response.getEmailAddress(), ERR_EMAIL_ADDR_INVALID);
+        assertEquals(DEFAULT_ACCOUNT_NAME, response.getName(), ERR_NAME_INVALID);
+        assertNotNull(response.getArn(), ERR_ARN_NULL);
+        assertNotNull(response.getCreateDate(), ERR_CREATE_DATE_NULL);
+        assertEquals(DEFAULT_ACCOUNT_ID, response.getId(), ERR_ID_NULL);
+        assertEquals(DEFAULT_CANONICAL_ID, response.getCanonicalId(), ERR_CANONICAL_ID_NULL);
+        assertNotNull(response.getCustomAttributes(), ERR_CUSTOM_ATTRIBUTES_NULL);
+    }
+
+    @Test
+    public void testGetAccountWithCredentials() throws Exception {
+        GetAccountRequestDTO getAccountRequestDTO = GetAccountRequestDTO.builder()
+                .accountName(DEFAULT_ACCOUNT_NAME)
+                .build();
+
+        accountServicesClient = new AccountServicesClient(getAccountAmazonHttpClient, basicAWSCredentials);
+        AccountData response = accountServicesClient.getAccount(getAccountRequestDTO).getAwsResponse();
+
+        assertEquals(DEFAULT_EMAIL_ADDR, response.getEmailAddress(), ERR_EMAIL_ADDR_INVALID);
+        assertEquals(DEFAULT_ACCOUNT_NAME, response.getName(), ERR_NAME_INVALID);
+        assertNotNull(response.getArn(), ERR_ARN_NULL);
+        assertNotNull(response.getCreateDate(), ERR_CREATE_DATE_NULL);
+        assertEquals(DEFAULT_ACCOUNT_ID, response.getId(), ERR_ID_NULL);
+        assertEquals(DEFAULT_CANONICAL_ID, response.getCanonicalId(), ERR_CANONICAL_ID_NULL);
+        assertNotNull(response.getCustomAttributes(), ERR_CUSTOM_ATTRIBUTES_NULL);
+        init();
+    }
+
+    @Test
+    public void testGetAccountWithInvalidRequest(){
+        GetAccountRequestDTO getAccountRequestDTO = GetAccountRequestDTO.builder().build();
+
+        assertThrows(SdkClientException.class, () -> {
+            getAccountMockClient.getAccount(getAccountRequestDTO).getAwsResponse();
+        }, "Expected SdkClientException");
+    }
+
+    @Test
+    public void testGetAccountWithInvalidRequest2(){
+        GetAccountRequestDTO getAccountRequestDTO = GetAccountRequestDTO.builder()
+                                                    .accountName(DEFAULT_ACCOUNT_NAME)
+                                                    .accountArn(DEFAULT_ARN_STR)
+                                                    .build();
+
+        assertThrows(SdkClientException.class, () -> {
+            getAccountMockClient.getAccount(getAccountRequestDTO).getAwsResponse();
+        }, "Expected SdkClientException");
+    }
+
+    /** Tests with Actual Vault **/
+    @Disabled
+    @Test
+    @SuppressWarnings( "deprecation" )
+    public void testGetAccountWithActualVault() {
+        //"D4IT2AWSB588GO5J9T00": "UEEu8tYlsOGGrgf4DAiSZD6apVNPUWqRiPG0nTB6"
```

Comment:
Maybe add another comment that these are the default admin keys for vault.
Along with a link to the file: https://github.com/scality/Vault/blob/development/7.10/tests/utils/admincredentials.json